docs/ShippingSimulator.md

```diff
@@ -40,6 +40,7 @@ The Shipping Simulator block **estimates the shipping fee** based on a zip code
 | ----------------------- | --------- | --------------------------------------------------------------------------------------------- | ------------- |
 | `skuId`                 | `String`  | ID of the current product SKU                                                                 | -             |
 | `seller`                | `String`  | ID of the product seller                                                                      | -             |
+| `sumAllValues`          | `Boolean` | If the product has gifts or attachments, you can choose whether the shipping information will be grouped by shipping type or showing the shipping prices for each of the items individually. | `false`       |
```

Comment:
Instead of creating a boolean prop, how about creating an enum?

We tend to stay away from boolean props as much as possible so that our component's APIs don't grow indefinitely.

For this use case, how about a `pricingMode` prop, with possible values: `individualItems` and `grouped`? Something along those lines.

Then in the code, `individualItems` would be the default case and `grouped` would be the behavior you've just implemented.

Reply:
@victorhmp,

Thanks for the sugestion.

I will implement these changes and will soon return.

Reply:
@victorhmp 

Corrections made.

Could you review it again, please?